src/components/filter.js

```diff
@@ -1,19 +1,31 @@
+import {getMarkup} from "../utils";
+import {filterElements} from "../data";
 
-const getFilterTemplate = ({name, count = 0, isChecked = false} = {}) => {
+const filterBlock = ({name, count = 0, isChecked = false} = {}) => {
```

Comment:
```suggestion
const filterBlock = ({name, count = 0, isChecked = false}) => {
```

См. https://github.com/htmlacademy-ecmascript/1006565-taskmanager-9/pull/4#discussion_r314962047. (уже не критично)